cleanlab/rank.py

```diff
@@ -652,3 +653,106 @@ def get_outlier_scores(
         return (outlier_scores, knn)
     else:
         return outlier_scores
+
+
+def get_ood_scores(
+    pred_probs: np.array,
+    confident_thresholds: Optional[np.ndarray] = None,
+    labels: Optional[np.ndarray] = None,
```

Comment:
Let's replace these 3 lines with:
```
pred_probs: np.array, *,
    labels: Optional[np.ndarray] = None,
    confident_thresholds: Optional[np.ndarray] = None,
```

Note I've changed the order of the latter two args, and also made them keyword only args, so that in the future we can swap their order again without breaking existing users' code.

Reply:
Good idea!